cmd/wasm-client/main.go

```diff
@@ -340,16 +342,28 @@ func (w *wasmClient) InvokeRPC(_ js.Value, args []js.Value) interface{} {
 
 	method, ok := w.registry[rpcName]
 	if !ok {
+		log.Errorf("RPC method '%s' not found in registry", rpcName)
 		return js.ValueOf("rpc with name " + rpcName + " not found")
 	}
 
 	go func() {
+		defer func() {
+			if r := recover(); r != nil {
+				errMsg := fmt.Sprintf("Panic in RPC call: %v", r)
+				log.Errorf("%s\n%s", errMsg, debug.Stack())
+				jsCallback.Invoke(js.ValueOf(errMsg))
+			}
```

Comment:
I'm not fully sure that we'll be able to recover from this as the actual daemons might have panicked, but I think it should be fine to handle it like this here either way.